File: app/src/main/java/org/michaelbel/moviemade/ui/modules/reviews/ReviewsMvp.java
package org.michaelbel.moviemade.ui.modules.reviews;

import com.arellomobile.mvp.MvpView;

import org.michaelbel.moviemade.annotation.EmptyViewMode;
import org.michaelbel.moviemade.data.dao.Review;

import java.util.List;

public interface ReviewsMvp extends MvpView {

    void setReviews(List<Review> reviews, boolean firstPage);

    void showError(@EmptyViewMode int mode);
}

File: app/src/main/java/org/michaelbel/moviemade/ui/modules/reviews/ReviewsMvp.kt
package org.michaelbel.moviemade.ui.modules.reviews

import com.arellomobile.mvp.MvpView
import org.michaelbel.moviemade.data.entity.Review
import org.michaelbel.moviemade.utils.EmptyViewMode

interface ReviewsMvp : MvpView {

    fun setReviews(reviews: List<Review>)

    fun setError(@EmptyViewMode mode: Int)
}